app/src/main/res/values/style.xml

```diff
@@ -1,29 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+
     <style name="SplashTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
a style with only one property does not make sense. Try to use the property directly